docs/cli-usage.md

```diff
@@ -1,4 +1,4 @@
-# CLI Usage
+# CLI Reference
```

Comment:
Did you mean to change this? Title of PR makes me think you wanted "CLI Usage" not "CLI Reference"

Reply:
After @brendongo’s PR #105 merged, he and I both realized there was already a doc page with CLI info (mostly exit codes). So we decided to combine the contents of Brendon’s PR with the CLI content that was already there.

Reply:
So should this be closed?

Reply:
This PR is to merge the two pages under this new name. I felt reference was better for having both the --help and exit codes